lib/ApiResource.php

```diff
@@ -64,11 +64,10 @@ public static function classUrl()
     }
 
     /**
-     * @return string The full API URL for this API resource.
+     * @return string The instance endpoint URL for the given class.
      */
-    public function instanceUrl()
+    public static function classInstanceUrl($id)
```

Comment:
I _think_ you could just call this `instanceUrl` as well. The disambiguation is kind of nice, but I'm not crazy about starting a convention to prefix static functions with `class*`.